.github/single-file-samples/main.exs

```diff
@@ -6,8 +6,8 @@ Application.put_env(:sample, Example.Endpoint,
 )
 
 Mix.install([
-  {:plug_cowboy, "~> 2.5"},
-  {:jason, "~> 1.0"},
+  {:plug_cowboy, "~> 2.7"},
+  {:jason, "~> 1.4"},
```

Comment:
Is there a reason to restrict this to version >= 2.7 / 1.4?

The existing definitions allow any version of plug_cowboy >= 2.5 < 3 and jason >= 1.0 < 2, so there should be no need to change this.

Reply:
Yikes ok, this is just me incorrectly assuming how the ~> specifier works. Sorry for the noise.